tests/articulation_tests.ts

```diff
@@ -274,7 +300,7 @@ function drawArticulations2(options: TestOptions, contextBuilder: ContextBuilder
   }
 
   // Helper function to justify and draw a 4/4 voice
-  Formatter.FormatAndDraw(ctx, staveBar4, notesBar4);
+  x += formatAndDrawToWidth(x, y, width, notesBar4, Barline.type.END, []);
```

Comment:
What is the aim of `x` it seems not to be used.

Reply:
It is the first parameter to formatAndDrawToWidth in line 285